src/modules/terraform.rs

```diff
@@ -84,17 +89,22 @@ fn get_terraform_workspace(context: &Context) -> Option<String> {
 }
 
 fn parse_terraform_version(version: &str) -> Option<String> {
-    // `terraform version` output looks like this
-    // Terraform v0.12.14
-    // With potential extra output if it detects you are not running the latest version
+    // `terraform version` or `tofu version` output looks like this
+    //   Terraform v0.12.14/OpenTofu v1.7.2
+    // with potential extra output if it detects you are not running the latest version
+    // finds the version by grabbing the first line of output, splitting by spaces, and
+    // then skipping the first word
+
     let version = version
         .lines()
         .next()?
-        .trim_start_matches("Terraform ")
-        .trim()
-        .trim_start_matches('v');
+        .split_whitespace()
+        .skip(1)
+        .collect::<Vec<&str>>()
+        .join(" ");
 
-    Some(version.to_string())
+    // skip the `v` at the beginning of the string
+    Some(version[1..].to_string())
```

Comment:
Please keep using `.trim_start_matches('v')`, or at least use `.get(1..)` instead to avoid potential panics.